### PR TITLE
#170947427 Configure coveralls

### PR DESCRIPTION
**Pull request by RusimbiPatrick**
#### What does this PR do?
Have coveralls configured
#### Description of Task to be completed?

- Have coverage s script to integrate with Travis CI
- Show coverage

#### How should this be manually tested?
After Cloning the repo, run `npm i `and then `npm test`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#170947427](https://www.pivotaltracker.com/story/show/170947427)
#### Screenshots (if appropriate)
Screen Shot 2020-02-05 at 17 56 14

#### Questions:
N/A

### CODE REVIEW

**Review by hakoemmy — CHANGES_REQUESTED** [review]
Good job @RusimbiP, please fix that ESLint issue 
But I don't see badge in the readme as well.

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: MtBpU46onlKMKsYFG96zsV4xf2Xejh0cq
```
Comment (hakoemmy) [inline .coveralls.yml]: Add a blank line on a new line.

**Review by hakoemmy — CHANGES_REQUESTED** [review]

File: .coveralls.yml (hunk 1 above)
Comment (RusimbiPatrick) [reply .coveralls.yml]: Copy that

**Review by hakoemmy — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+[![Coverage Status](https://coveralls.io/repos/github/andela/octopus-bn-backend/badge.svg?branch=ch-configure-coveralls-170947427)](https://coveralls.io/github/andela/octopus-bn-backend?branch=ch-configure-coveralls-170947427)
 Barefoot Nomad - Making company travel and accomodation easy and convinient.
 =======
 [![](https://img.shields.io/badge/Reviewed_by-Hound-a873d1.svg)](https://houndci.com)
```
Comment (AbdoulNuru) [inline README.md]: @hakoemmy can these badges be arranged in order because As I can see one badge is above the title and another is below

**Review by AbdoulNuru — CHANGES_REQUESTED** [review]

**Review by AbdoulNuru — APPROVED** [review]

**Review by blaise82 — APPROVED** [review]

**Review by kcharles52 — CHANGES_REQUESTED** [review]
Please restore the modules that you deleted and are needed within the project

**Review by RusimbiPatrick — COMMENTED** [review]
> Please restore the modules that you deleted and are needed within the project

@kcharles52 Changes have been made

**Review by izzett222 — APPROVED** [review]

**Review by kcharles52 — APPROVED** [review]

**Pull request merged**